src/tribler-gui/tribler_gui/core_manager.py

```diff
@@ -59,7 +60,8 @@ def on_core_finished(self, exit_code, exit_status):
             exception_msg = "The Tribler core has unexpectedly finished with exit code %s and status: %s!" % \
                             (exit_code, exit_status)
             if self.core_traceback:
-                exception_msg += "\n\n%s" % self.core_traceback
+                exception_msg += "\n\n%s\n(Timestamp: %d, traceback timestamp: %d)" % \
+                                 (self.core_traceback, int(round(time.time() * 1000)), self.core_traceback_timestamp)
```

Comment:
where is `self.core_traceback_timestamp` coming from?

Reply:
Oh whoops! Seems that I removed one variable too much :'(